MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -162,7 +163,36 @@ public class ServerConfiguration : BaseApplicationConfiguration
         public bool SkipDeserializationForBasicTypes { get; set; }
 
         public string ServerName { get; set; }
-        public string BaseUrl { get; set; }
+        public string BaseUrl
+        {
+            get => _baseUrl;
+            set
+            {
+                _baseUrl = value;
```

Comment:
Why? Just work with value intill it's set properly.

Reply:
Because I'm modifying it in-place later.

Reply:
You can modify `value` afaik

Reply:
You can modify `value` afaik

Reply:
Because if the function gets aborted while it's unfinished (or the field checked) the field will contain an "invalid" value. It's just bad practice imo.